v2/pkg/engine/postprocess/rewrite_op_names.go

```diff
@@ -0,0 +1,44 @@
+package postprocess
+
+import (
+	"fmt"
+	"github.com/wundergraph/graphql-go-tools/v2/pkg/engine/resolve"
+	"strings"
+)
+
+// rewriteOpNames is a processor to rewrite operation names when they are propagated downstream.
+type rewriteOpNames struct {
```

Comment:
please be more specific what it is doing, e.g. append fetch id to the propagated operation name